pkgs/tools/misc/yubikey-neo-manager/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, makeWrapper, python27Packages, pyside, pycrypto
+, libykneomgr, yubikey-personalization, libu2f-host }:
+
+python27Packages.buildPythonPackage rec {
+  namePrefix = "";
+  name = "yubikey-neo-manager-${version}";
+  version = "1.4.0";
+  src = fetchurl {
+    url = "https://developers.yubico.com/yubikey-neo-manager/Releases/${name}.tar.gz";
+    sha256 = "1isxvx27hk0avxwgwcwys2z8ickfs816ii1aklvmi09ak1rgrf1g";
+  };
+
+  propagatedBuildInputs = [ pyside pycrypto ];
+  patches = [ ./fix-pyside-requirement.diff ];
+
+  # XXX can we fix ctypes "load_library()" instead?
```

Comment:
Is this of general interest for packaging python ffi modules? If so, maybe open a separte issue for it.